jerry-core/debugger/debugger.h

```diff
@@ -376,6 +376,7 @@ typedef struct
 typedef struct
 {
   uint8_t type; /**< type of the message */
+  uint8_t start; /**< starting point (0 - max_depth)*/
```

Comment:
This should be:
`uint8_t min_depth[sizeof (uint32_t)]; /**< minimum depth */`

Similar to:
`uint8_t max_depth[sizeof (uint32_t)]; /**< maximum depth (0 - unlimited) */`